uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -166,10 +168,9 @@ public void connect(String clientDeviceId, String componentId, String brokerId)
                      .createMqttConnection(request, connectionEvents);
     }
 
-    @Then("connection for device {string} is successfully established within {int} {word}")
-    public void validateConnect(String clientDeviceId, int value, String unitOfMeasure) {
-        //@TODO Implement method
-        throw new RuntimeException("Method validateConnect is not implemented");
+    @And("I set MQTT timeout {int}")
```

Comment:
Please add javadoc comment

Reply:
This will help set the timeout in the future